aws/tf/sra.tf

```diff
@@ -25,9 +25,12 @@ module "SRA" {
   // Account - Unity Catalog:
   metastore_id            = null // Metastore configuration - leave null if there is no existing regional metastore, does not create a root storage location
   metastore_name          = join("", [var.resource_prefix, "-", var.region, "-", "uc"])
-  data_bucket             = null // REQUIRED - Existing S3 bucket name (e.g. data-bucket-s3-test)
   workspace_catalog_admin = null // REQUIRED - Workspace specific catalogs are created, this user will become an admin of that catalog (e.g. firstname.lastname@company.com)
-  external_location_admin = null // REQUIRED - Read-only external location is created, this user will become an admin of that exteranl location (e.g. firstname.lastname@company.com)
+
+  // Account - Unity Catalog - Data:
+  enable_read_only_external_location = false // Creates a read-only external location and corresponding IAM role, based on the value of the data_bucket variable
+  read_only_data_bucket              = null  // Accepts an S3 path (e.g. "s3://data-bucket")
```

Comment:
The example in the comment references the bucket URL, but in file `uc_external_location.tf`, this variable has the `s3://` prefix prepended.  Just the bucket name works so _(e.g. "<del>s3://</del>data-bucket")_ --> _(e.g. "data-bucket")_